docker/base-dev.Dockerfile

```diff
@@ -10,51 +10,41 @@ ENV PYTHONFAULTHANDLER=1 \
     PIP_DISABLE_PIP_VERSION_CHECK=on \
     POETRY_HOME=/root/.local
 
-ENV ZENML_DEBUG=true
-ENV ZENML_ANALYTICS_OPT_IN=false
-ENV PATH="$POETRY_HOME/bin:$PATH"
-
 RUN apt-get update && \
-  apt-get install --no-install-recommends -q -y software-properties-common && \
-  add-apt-repository ppa:deadsnakes/ppa && \
-  add-apt-repository ppa:maarten-fonville/protobuf && \
-  apt-get update && \
   apt-get install --no-install-recommends -q -y \
   build-essential \
   ca-certificates \
   libsnappy-dev \
   protobuf-compiler \
   libprotobuf-dev \
-  python3.7-dev \
-  python3.7-venv \
-  wget \
+  python3 \
```

Comment:
So which version of python does this install right now? And will this ever change? I'm not very experienced with aptitude, some way to fixate the version?

Reply:
@jwwwb it depends on the base image. For `ubuntu:20.04`, it installs the python version that ships with Ubuntu 20.04 (currently, `3.8.10`). The cuda images inherit from an Ubuntu 18.04 base image which includes python `3.6.9` (I know, not ideal). We could freeze the python version too, yes, by using something like `python:3.7-slim` as our base image.